### PR TITLE
Ansible with Openstack support

### PR DESCRIPTION
**Pull request by brainstorm**
Moving this set of recipes from bioconda over conda-forge, related with https://github.com/bioconda/bioconda-recipes/pull/5336 and should supersede PR https://github.com/conda-forge/staged-recipes/pull/3353.

@souravsingh, I've introduced the suggestions from @ocefpaf and @scopatz over here too.

### CODE REVIEW

**Comment by brainstorm** [comment]
Ok, someone can help me transition those recipes from bioconda environment/channel to conda-forge? Seems I'm failing bad on all CI platforms where I used to be ok over bioconda :-S

**Comment by souravsingh** [comment]
I can check and help with the migration of recipes.

**Comment by brainstorm** [comment]
@souravsingh Can you shed some light into this TravisCI error? Have you seen it before while working w/conda-forge vs bioconda?: https://travis-ci.org/conda-forge/staged-recipes/builds/261066807

File: recipes/dogpile.cache/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  md5: 66e0a6cae6c08cb1ea25f89d0eadfeb0
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - six
```
Comment (isuruf) [inline recipes/dogpile.cache/meta.yaml]: Needs `setuptools` here

**Comment by brainstorm** [comment]
Ugh, when I was packaging that for bioconda, the common warning message was to remove `setuptools` from the mix... I'll add it back then.

**Comment by brainstorm** [comment]
Now the CI log output gets trucated O_o'''

Any ideas on what might be causing this?

**Comment by isuruf** [comment]
Can you break this PR into a few PRs? CIs fail when there is log output > 4M.

**Comment by brainstorm** [comment]
Thanks for the feedback @isuruf but I'm a bit uncertain that would help since the same PR succeeded (all green, no issues) on TravisCI instance from bioconda:

https://github.com/bioconda/bioconda-recipes/pull/5336

Isn't it possible that it's something else going on? Recursive inclusion/import or something else I'm missing?

**Comment by isuruf** [comment]
In bioconda, the output is suppressed.

**Comment by brainstorm** [comment]
Closing this PR in favor of all of the referred/splitted ones.

**Pull request closed without merging**